reflex/app.py

```diff
@@ -519,13 +520,14 @@ def get_frontend_packages(self, imports: Dict[str, str]):
         """
         page_imports = {
             i
-            for i in imports
+            for i, tags in imports.items()
             if i not in compiler.DEFAULT_IMPORTS.keys()
             and i != "focus-visible/dist/focus-visible"
             and "next" not in i
```

Comment:
maybe out of scope for this PR, but I think these lines could be cleaned up by marking `focus-visible/dist/focus-visible` over in DEFAULT_IMPORTS with `install=False`, and changing `"next" not in i` to `"not i.startswith("next/")`.

i haven't tried this, but ultimately i _should_ be ignoring any dep that appears in the templated `package.json`, as we don't want to overwrite those pins.

Reply:
Yeah it's probably a good idea to clean up this part and avoid "exception" as much as possible in how we select "to-be-installed" packages